vm/src/serde/deserialize_program.rs

```diff
@@ -139,9 +139,9 @@ pub struct Location {
     pub start_col: u32,
 }
 
-#[derive(Deserialize, Debug, PartialEq, Eq)]
+#[derive(Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct DebugInfo {
-    instruction_locations: HashMap<usize, InstructionLocation>,
+    pub instruction_locations: HashMap<usize, InstructionLocation>,
```

Comment:
It would be preferable to not make things public. Could you implement some `fn get(&self, pc: usize) -> &InstructionLocation` method or something like that instead?

Reply:
I have made it private again. Instead of `fn get(&self, pc: usize) -> &InstructionLocation` I have added `pub fn get_instruction_locations(&self) -> HashMap<usize, InstructionLocation>` because I need to iterate over them. This function returns a clone so it won't modify the existing data and it should be safe.